examples/render-helm-chart-kustomize-inline-values/README.md

```diff
@@ -0,0 +1,83 @@
+# render-helm-chart: Kustomize Inline Values
+
+### Overview
+
+This example demonstrates how to declaratively invoke the `render-helm-chart`
+function with kustomize using the `valuesInline` field.
+
+### Function invocation
+
+To use the function with kustomize, you can specify the `functionConfig`
+in your kustomization's `generators` field. This example uses inline values
+to use instead of the default values accompanying the chart:
+
+kustomization.yaml:
+```yaml
+generators:
+- |-
+  apiVersion: fn.kpt.dev/v1alpha1
+  kind: RenderHelmChart
+  metadata:
+    name: demo
+    annotations:
+      config.kubernetes.io/function: |
+        container:
+          network: true
+          image: gcr.io/kpt-fn/render-helm-chart:unstable
+  helmCharts:
+  - name: ocp-pipeline
+    namespace: mynamespace
+    version: 0.1.16
+    repo: https://bcgov.github.io/helm-charts
+    releaseName: moria
+    valuesInline:
+      releaseNamespace: ""
+      rbac:
+        create: true
+        rules:
+          - apiGroups: [""]
+            verbs: ["*"]
+            resources: ["*"]
+```
+
+Then, to build the kustomization with kustomize v4:
+
+```shell
+kustomize build --enable-alpha-plugins --network .
+```
+
+### Expected result
+
+You should also be able to find `def releaseNamespace = ""` in your output, as
```

Comment:
> You should also be able to find `def releaseNamespace = ""` in your output

What's this? I'm not following

Reply:
The output contains the following substring:

```
  strategy:
    jenkinsPipelineStrategy:
      jenkinsfile: |-
        def helmName = "helm-v3.1.0-linux-amd64.tar.gz"
        def chartName = "metadata-curator"
        def chartRepo = "http://bcgov.github.io/helm-charts"
        def releaseName  = "mc"
        def releaseNamespace = ""
```

Without the inline values setting `releaseNamespace: ""`, the output would instead have the following: 

```
  strategy:
    jenkinsPipelineStrategy:
      jenkinsfile: |-
        def helmName = "helm-v3.1.0-linux-amd64.tar.gz"
        def chartName = "metadata-curator"
        def chartRepo = "http://bcgov.github.io/helm-charts"
        def releaseName  = "mc"
        def releaseNamespace = "mc"
```

So finding the line `def releaseNamespace = ""` in the output indicates that the correct values are being used.

Reply:
I updated the wording slightly in the README. Does it help?